nomad/worker.go

```diff
@@ -79,9 +83,10 @@ type Worker struct {
 // NewWorker starts a new worker associated with the given server
 func NewWorker(ctx context.Context, srv *Server) (*Worker, error) {
 	w := &Worker{
-		srv:   srv,
-		start: time.Now(),
-		id:    uuid.Generate(),
+		srv:               srv,
+		start:             time.Now(),
+		id:                uuid.Generate(),
+		enabledSchedulers: srv.GetSchedulerWorkerConfig().EnabledSchedulers,
```

Comment:
For my own edification, why do we not need to call `copy()` on this slice here?

Reply:
Addressed by making GetSchedulerWorkerConfig have a copy